homeassistant/components/homematic/entity.py

```diff
@@ -234,6 +241,8 @@ def state(self):
     def state_attributes(self):
         """Return the state attributes."""
         attr = self._variables.copy()
+        if self._duty_cycle is not None:
+            attr["DutyCycle"] = self._duty_cycle
```

Comment:
Since `DutyCycle` is used more than once, please add a variable for this at the top and reuse it in the code. Same for `DUTY_CYCLE` further down.